src/sentry/static/sentry/app/components/sidebar/broadcasts.jsx

```diff
@@ -79,65 +73,59 @@ const Broadcasts = createReactClass({
         });
         this.startPoll();
       });
-  },
+  }
 
   startPoll() {
     this.poller = window.setTimeout(this.fetchData, POLLER_DELAY);
```

Comment:
Do you need to `bind()` this function?

Reply:
Good catch, it needs to turn into an arrow function.

Reply:
Yeah `fetchData` did, updated and checked any others.